plugins/obs-x264/obs-x264.c

```diff
@@ -109,6 +117,10 @@ static void obs_x264_defaults(obs_data_t *settings)
 	obs_data_set_default_string(settings, "preset", "veryfast");
 	obs_data_set_default_string(settings, "profile", "");
 	obs_data_set_default_string(settings, "tune", "");
+	/* 2020-01-19:  the difference between UTC and TAI is 37 seconds */
+	obs_data_set_default_int(settings, "utc_to_tai", 37);
```

Comment:
The IETF maintains a file we could download and parse (in regular intervals): https://www.ietf.org/timezones/data/leap-seconds.list

On Linux you could use e.g. `ntp_gettime` to get the current offset as well. Not sure if Windows has an equivalent. `std::chrono::tai_clock` and associated features are C++20 and I think we're still targeting C++17 for now.

Reply:
tzdata (zoneinfo) on Linux also exposes this leap second information in `/usr/share/zoneinfo/leap-seconds.list` which we could use instead of a remote file (assuming the user keeps their distro up to date). Windows does provide access to the "leap indicator" from NTP but this requires tracking state ourselves and is unreliable if OBS isn't run at least once a day so it's of no practical use. Unsure about macOS.

Reply:
On macOS it's `/usr/share/zoneinfo/leapseconds`. So that leaves everyone's favourtie non-POSIX operating system...